lib/outdated.js

```diff
@@ -16,7 +16,7 @@ packages.
 
 module.exports = outdated
 
-outdated.usage = 'npm outdated [[<@scope>/]<pkg> ...]'
+outdated.usage = ('outdated', '\nnpm outdated [[<@scope>/]<pkg> ...]', '[--json][--long][--parseable][--global][--depth]')
```

Comment:
Did you mean for this to be calling the `lib/install/usage.js` function?  As you have this written, it's just a parenthesized group of expressions, which resolves to the last expression.

Reply:
@isaacs - probably? I was following the pattern I found on `install` because I wasn't very confident with what it _should_ do.

I just wanted to make it easier to see what the options were without having to go into the code base like I did.